README.md

```diff
@@ -19,25 +18,98 @@ The bridge itself proxies those services to the cloud with the proper authentica
 
 ### What you’ll need
 * JDK 1.7
 * Maven 3.0.5+
 
 ### Build with maven
-Integration Core is compatible with Apache Maven 3.0.5 or above. If you don’t already have Maven installed you can
-follow the instructions at maven.apache.org.
+Integration Core is compatible with Apache Maven 3.0.5 or above. If you don’t already have Maven installed you can follow the instructions at maven.apache.org.
 
 To start from scratch, do the following:
 
-1. Clone the source repository using Git:
-   `git clone https://github.com/SymphonyOSF/App-Integrations-Core.git`
-2. cd into _App-Integrations-Core_
-3. Build using maven:
-   `mvn clean install`
+1. Build the _App-Integrations-Core_ dependencies, on this order (so you have them in your Maven local repository):
+
+> 1. [_App-Integrations-Commons_](https://github.com/symphonyoss/App-Integrations-Commons) 
+> 2. [_App-Integrations-Universal_](https://github.com/symphonyoss/App-Integrations-Universal)
+> 3. [_App-Integrations-Github_](https://github.com/symphonyoss/App-Integrations-Github)
+> 4. [_App-Integrations-Jira_](https://github.com/symphonyoss/App-Integrations-Jira)
+> 5. [_App-Integrations-Salesforce_](https://github.com/symphonyoss/App-Integrations-Salesforce)
+> 6. [_App-Integrations-Trello_](https://github.com/symphonyoss/App-Integrations-Trello)
+> 7. [_App-Integrations-Zapier_](https://github.com/symphonyoss/App-Integrations-Zapier)
+
+2. Clone the source repository using Git: `git clone git@github.com:symphonyoss/App-Integrations-Core.git`
+3. cd into _App-Integrations-Core_
+4. Build using maven: `mvn clean install`
+
+# Configuring the project
+Here are the initial steps to get your project configured to run using the Intellij IDEA IDE.
 
-Notes: If you have no access to Symphony Artifactory you should build the _App-Integrations-Commons_ previously.
\ No newline at end of file
+1. Import this project into your IDE as a ``maven`` project and be sure to choose JDK 1.7 to run it with.
+2. Import any other Integration projects the same way as above (like, let's say, App-Integrations-Github or App-Integrations-Commons), but those are not required.
+3. Copy [this file](docs/configuration/idea/IntegrationBridgeApplication.xml) to your "project source folder"/.idea/runConfigurations (feel free to create the last folder if you don't have it yet).
+4. Go to ``Run > Edit Configurations...`` and select check the one called "IntegrationBridgeApplication"
+5. Check that the referenced folders do exist, they should all be pointing to ``docs/configuration/boot/`` folders, to exemplify the structure you need
+6. Obtain valid, PKCS#12 user certificates to your POD and copy those to ``docs/configuration/boot/certs``, you'll need one for each integration.
+7. Configure valid addresses to connect the application to on the file [application.yaml](docs/configuration/boot/application.yaml)
+8. Run ``IntegrationBridgeApplication`` from the "Run" menu and we're done!
```

Comment:
In order for this to work, you'll need to have built already all the other modules, or else you won't have all the jars required to get the integration-core running.
It's like this today because we have to add all dependencies from all other modules that will be exposed through the integration-web module. He'll expose a servlet, and he must be aware of all other integrations to work properly, it's not dynamic.
I'll add a screenshot or log snippet demonstrating what people should look for when it run successfully.